xain/grpc/coordinator.py

```diff
@@ -0,0 +1,48 @@
+import time
+from concurrent import futures
+
+import grpc
+
+from xain.grpc import coordinator_pb2, coordinator_pb2_grpc
+
+_ONE_DAY_IN_SECONDS = 60 * 60 * 24
+
+
+class Coordinator(coordinator_pb2_grpc.CoordinatorServicer):
+    def __init__(self, required_participants=10):
```

Comment:
I am not sure but it might be useful/easier to grasp to use similar concepts as in the FL package?

For example using something like: `num_participants=100` and `C=0.1` and derive `required_participants` from those numbers (`num_participants * C`).

What do you think?

Reply:
I agree

Reply:
I agree but what I'm trying at this point is to just get all the necessary grpc communication done before starting to add any business logic.

This was added more for testing purposes. In a next iteration when we start adding businees logic we could make the APIs the same as in the fl module

Reply:
Ok, got it. So lets resolve this for now.

Reply:
Ok for me